docs/source/configuration/advanced_configuration.md

```diff
@@ -60,8 +60,8 @@ bucket_name: "my_s3_bucket"
 key_prefix: "my/key/prefix/"
 
 datasets:
-    csv: "pandas.CSVDataSet"
-    spark: "spark.SparkDataSet"
+    csv: "pandas.CSVDataset"
```

Comment:
These dataset names won't work unless the user upgrades `kedro-datasets`; do we need some sort of communication around this? Does it need to be in the docs?

Reply:
This is going into `develop`, I think we should add some messaging in the `main` branch soon so users already know this is happening before `0.19.0` and `2.0.0` for datasets.